src/argus/notificationprofile/media/__init__.py

```diff
@@ -52,18 +52,17 @@ def api_safely_get_medium_object(media_slug):
 def send_notification(destinations: Iterable[DestinationConfig], *events: Iterable[Event]):
     if not events:
         return
-    sent = False
     media = get_notification_media(destinations)
     # Plugin expects queryset...
     ids = (dest.id for dest in destinations)
     qs = DestinationConfig.objects.filter(id__in=ids)
+    media_count = len(media)
     for event in events:
-        LOG.info('Notification: sending event "%s"', event)
+        LOG.info('Notification: sending event "%s" to %i mediums', event, media_count)
         for medium in media:
-            sent = medium.send(event, qs) or sent
-    if not sent:
-        LOG.info("Notification: Could not send notification, nowhere to send it to")
-        # TODO Store error as incident
+            sent = medium.send(event, qs)
+            if sent:
+                LOG.info('Notification: sent event "%s" to "%s"', event, medium.MEDIA_SLUG)
```

Comment:
```suggestion
                LOG.info('Notification: sent event "%s" to "%s"', event, medium.MEDIA_NAME)
```

Not very important, just a bit nicer

Reply:
"nice" is subjective :) I prefer slugs since I can be certain there are no invisible characters in them. WYSIWYG.